.kokoro/deps-finder.sh

```diff
@@ -0,0 +1,143 @@
+# limitations under the License.
+
+# Compare shared-dependencies version in a given dependency to desired version.
+# Return 0 for success, 1 for failed to find POM, 2 for shared-deps not present, 3 for incorrect version.
+
+# Options to run:
+# 0 Arguments: Test the latest commit (with a dependency update) for the latest version of shared-deps.
+# 1 Argument: Input artifactId:version to test the given dependency for the latest version of shared-deps.
+# 2 Arguments: Input artifactId:version to test the given dependency for a specific version of shared-dependencies
+# (only input a version number for the second argument).
+
+# Get the directory of the build script
+scriptDir=$(realpath $(dirname "${BASH_SOURCE[0]}"))
+
+# include common functions
+source ${scriptDir}/common.sh
+source ${scriptDir}/shared-deps-helper.sh
+
+# Pass in dependency to script.
+# Dependency should be of the form artifactId:version
+
+function checkLibrary() {
+  # Should be of the format artifactId:version
+  clientLibrary=$1
+
+  # If no dependency was passed straight to the script, check our most recent commit for a dep version update.
+  if [[ -z ${clientLibrary} ]]; then
+    clientLibrary=$(getLibraryUpdate)
+    # We did not successfully find a valid commit with a dependency. Return success - nothing to check here!
+    if ! [[ $? -eq 0 ]]; then
+      echo "Commit message does not have correct formatting. Assuming no error."
+      return 0
+    fi
+  else
+    # Check if dependency contains 'google-cloud-'. If it does not, it's already valid!
+    if [[ -z $(echo ${clientLibrary} | grep "google-cloud-") ]]; then
+      echo "${clientLibrary} does not require shared-dependencies criteria. Returning success."
+      return 0
+    fi
+  fi
+
+  # Should just be the desired version for shared-deps, e.g. "0.8.3"
+  latestSharedDeps=$2
+
+  if [[ -z ${latestSharedDeps} ]]; then
+    latestSharedDeps=$(getLatestSharedDepsVersion)
+    if [[ -z ${latestSharedDeps} ]]; then
+      echo "Unable to find correct version of shared-dependencies."
+      return 1
+    fi
+  fi
+
+  # Grab last part of dependency artifactId:version input.
+  # Remove any alphabetical characters at the end (e.g. -alpha, -beta, etc)
+  depVersion=${clientLibrary/*:/}
+  depVersion=${depVersion/-*/}
+
+  pomLocation="/pom.xml"
+
+  # The naming of these libraries does not follow the rules. As such, we've added in special cases for them.
+  # depArtifactId => the ID inside of the Github link for the dependency.
+  case ${clientLibrary} in
+  *"google-cloud-bigtable-bom"*)
+    depArtifactId="java-bigtable"
+    pomLocation="/google-cloud-bigtable-deps-bom/pom.xml"
+    ;;
+  *"google-cloud-core-bom"*)
+    # This is an exceptional case which does not require shared-dependencies! Return true automatically.
+    echo "Found exception google-cloud-core-bom!"
+    return 0
+    ;;
+  *"google-cloud-build-bom"*)
+    depArtifactId="java-cloudbuild"
```

Comment:
This isn't an artifactId, it's the repo name.

Reply:
Fixed in the next version!